### PR TITLE
Added zh_cn course file and a Golang course.

### PR DESCRIPTION
**Pull request by GintasS**
## What does this PR do?
Added resources.

### Description
Added a course zh_cn file.
Added first Golang cource in it.

### Why is this valuable (or not)?
This course is free!

### How do we know it's really free?
It is provided freely by Microsoft on their website.

### For book lists, is it a book? For course lists, is it a course? etc.
It is a course.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [x] Search for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists  in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

## Followup

- Check the output of Travis-CI for linter errors!

### CODE REVIEW

**Comment by eshellman** [comment]
remember to add a link in the Readme

**Comment by GintasS** [comment]
> remember to add a link in the Readme

Fixed, let me know if the name is correct in the readme, since there are a lot of different Chinese languages (variations?).

**Comment by eshellman** [comment]
Hmmmm. I guess I didn't think through the spoken vs. written language issues. the zh_CN locale is usually for Chinese (simplified), while zh_HK and zh_TW are Chinese (traditional). But the dominant spoken languages are Mandarin and Cantonese. So courses may need to be separated by spoken languages while books and interactive tutorials would be separated by the simplified vs. traditional. so I think your original suggestion to create an interactive tutorial page is the better option. currently we have a single chinese page bor books, labeled [Chinese / 中文](books/free-programming-books-zh.md) so maybe we should do the same for interactive tutorials, and only divide by local when there are more items to divide. We have a HOWTO in traditional characters - the label is modified by the rendering of HOWTO.

**Comment by GintasS** [comment]
So, should I create free-programming-interactive-tutorials-**zh**.md and place this Golang course there?

**Comment by eshellman** [comment]
yes.

**Comment by GintasS** [comment]
Fixed.

File: README.md [hunk 1]
```diff
@@ -96,6 +96,7 @@ Please read [CONTRIBUTING](/CONTRIBUTING.md). If you're new to GitHub, [welcome]
 
 ### Interactive Programming Resources
 
++ [Chinese / 中文](courses/free-courses-zh_cn.md)
```
Comment (eshellman) [inline README.md:99]: it's in more/free-programming-interactive-tutorials-ru.md now

File: more/free-programming-interactive-tutorials-zh.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+### Index
+
+* [Golang](#golang)
+
```
Comment (eshellman) [inline more/free-programming-interactive-tutorials-zh.md:4]: linter wants another space here

**Review by GintasS — COMMENTED** [review]
Fixed, thanks!

**Pull request merged**